spec/models/book_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+require 'faker'
+
+describe Book, type: :model do
+  subject(:book) { build(:book) }
+  # Validation for each attr
```

Comment:
We try not to use comments except if something very difficult to understand or a TODO